source/includes/_webhooks.md

```diff
@@ -24,29 +24,63 @@ It is advisable to have tolerance for delivering a callback maximum up to ten mi
 
 Our implementation follows [Stripe's specification](https://stripe.com/docs/webhooks/signatures)
 
-## Incidents
+`dont_disturb_mode` user setting in mobile application has no influence on these events.
 
-This is a definition of an incident payload sent via webhook.
+## Incident events
 
-### parameters
+Webhook will receive all events related to incidents.
+
+Incidents have the following lifecycle:
+
+```
+created -> dismissed -> resolved
```

Comment:
You can always reopen the incident. I am not sure how to describe it here. BTW can you please check how this will look like from sending a webhook perspective when some reopens the incident? Do we send webhook again ?

Reply:
Should I reflect other actions like snooze and escalate here? If we will describe them here when we should send such webhook.

Right now if user reopens an incident we will not send anything, but if it will be resolved after, we will send a second `incident_resolved` event to the webhook. So it looks like it will be inconsistent if we omit `incident_reopened` event, we should implement sending for such events.

Reply:
Let's define for not just two event types - `created` -> `closed` . We don't have a clear definition of dismissed and resolved now and we might get into some issues with other states and transition .. e.g from dismissed you can get back to contacting which might be confusing .